docs/installation/deployment.rst

```diff
@@ -103,44 +110,46 @@ Additionally, there are some configurations that may commonly need to be set in
 
 The full set of configuration options are documented in :ref:`schemaregistry_config`.
 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+===================================
 Don't Modify These Storage Settings
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+===================================
 
 |sr| stores all schemas in a Kafka topic defined by ``kafkastore.topic``. Since this Kafka topic acts as the commit log for |sr| database and is the source of truth, writes to this store need to be durable. |sr| ships with very good defaults for all settings that affect the durability of writes to the Kafka based commit log. Finally, ``kafkastore.topic`` must be a compacted topic to avoid data loss. Whenever in doubt, leave these settings alone. If you must create the topic manually, this is an example of proper configuration:
 
 .. sourcecode:: bash
 
   # kafkastore.topic=_schemas
-    bin/kafka-topics --create --zookeeper localhost:2181 --topic connect-configs --replication-factor 3 --partitions 1 --config cleanup.policy=compact
+    bin/kafka-topics ==create ==zookeeper localhost:2181 ==topic connect-configs ==replication-factor 3 ==partitions 1 ==config cleanup.policy=compact
```

Comment:
@joel-hamill remember to change these back (e.g., `--create`, `--zookeeper` etc. )